CollaborationDemo/Views/Example/Form.cshtml

```diff
@@ -0,0 +1,16 @@
+@{
+    ViewData["Title"] = "Comment Form";
+}
+
+<h1>Questions, Comments & Suggestions</h1>
+
+<p> Please give us your feedback on our app!</p>
+
+<form method="POST">
```

Comment:
Wow, this is looking great so far! I think this is a great start to our feedback form. 
It would probably be good for us to start thinking about validation. What do you think of adding the required keyword to these form inputs for now, so we know for sure these will have values.  Later down the line we can revisit and refine it on a different branch?

Reply:
That's good thinking.  We can add more validation to the controllers later on.  On it!